R/Simulations-class.R

```diff
@@ -204,7 +205,8 @@ Simulations <- function(fit,
     slots = c(
       rho_est = "numeric",
       sigma2w_est = "numeric",
-      fit_biomarker = "list"
+      fit_biomarker = "list"#,
+   #   stop_report = "matrix"
```

Comment:
same here please remove `stop_report` here and below because this slot is already in the parent class `Simulations`